packages/auth/src/authClients/supabase.ts

```diff
@@ -1,30 +1,37 @@
-import type {
-  SupabaseAuthUser,
-  SupabaseClient as Supabase,
-  SupabaseAuthResponse,
-} from '@supabase/supabase-js'
+import { Session, User, Provider } from '@supabase/gotrue-js/dist/main/lib/types'
```

Comment:
@Tobbe and I wondered why it was that he had to dig so "deep" into these supabase libs to get at the types.

Reply:
I'll ask the supabase people if they can do a cleaner export of those types. But I can fix that in a follow-up PR.